ng/src/web/pages/usersettings/usersettingspage.js

```diff
@@ -0,0 +1,1064 @@
+/* Greenbone Security Assistant
+*
+* Authors:
+* Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+*
+* Copyright:
+* Copyright (C) 2017 Greenbone Networks GmbH
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React from 'react';
+import {Col} from 'glamorous';
+import glamorous from 'glamorous';
+
+import _ from 'gmp/locale.js';
+import {parse_yesno, YES_VALUE} from 'gmp/parser.js';
+import {is_defined, is_empty} from 'gmp/utils.js';
+
+import HelpIcon from '../../components/icon/helpicon.js';
+import EditIcon from '../../components/icon/editicon.js';
+
+import IconDivider from '../../components/layout/icondivider.js';
+import Layout from '../../components/layout/layout.js';
+import Section from '../../components/section/section.js';
+
+import DetailsLink from '../../components/link/detailslink.js';
+
+import Loading from '../../components/loading/loading.js';
+
+import Table from '../../components/table/simpletable.js';
+import TableBody from '../../components/table/body.js';
+import TableData from '../../components/table/data.js';
+import TableRow from '../../components/table/row.js';
+
+import PropTypes from '../../utils/proptypes.js';
+
+import withCapabilities from '../../utils/withCapabilities.js';
+import withGmp from '../../utils/withGmp.js';
+
+import SettingsDialog from './dialog.js';
+
+const CA_CERT_ID = '9ac801ea-39f8-11e6-bbaa-28d24461215b';
+
+const SEVERITY_CLASSES = [
+  {id: 'nist', name: 'NVD Vulnerability Severity Ratings'},
+  {id: 'bsi', name: 'BSI Schwachstellenampel (Germany)'},
+  {id: 'classic', name: 'OpenVAS Classic'},
+  {id: 'pci-dss', name: 'PCI-DSS'},
+];
+
+const Heading = glamorous.h4({marginBottom: '5px'});
+
+const ToolBarIcons = ({onEditSettingsClick}) => (
+  <Layout flex>
+    <IconDivider>
+      <HelpIcon
+        page="my_settings"
+        title={_('Help: My Settings')}
+      />
+      <EditIcon
```

Comment:
We should check the capability for edit settings here: `capabilities.mayEdit('settings')`